exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -50,21 +50,19 @@ type OperationMap = Map<number, Operation>;
 type OptimisticDependencies = Map<number, Dependencies>;
 type DependentOperations = Record<string, number[]>;
 
-export interface CacheExchangeOpts {
-  updates?: Partial<UpdatesConfig>;
-  resolvers?: ResolverConfig;
-  optimistic?: OptimisticMutationConfig;
-  keys?: KeyingConfig;
-  schema?: IntrospectionData;
-  storage?: StorageAdapter;
-}
-
-export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
-  forward,
-  client,
-  dispatchDebug,
-}) => {
-  const store = new Store(opts);
+export type CacheExchangeOpts = {
```

Comment:
Is there a point to wrapping this in `Partial` rather than keeping each field as optional like before?